rfcs/0011-releases.md

```diff
@@ -0,0 +1,73 @@
+# Releasing Composable Node and Related Components
+
+## 1. Abstract
+
+Given the complexity of the number of components and teams involved a standard process is needed to release Composable node and related components.
+
+## 2. Goals
+
+1. Define release numbering convention.
+2. Define release process steps.
+
+## 3. Method
+
+### 3.1. Release Numbers
+
+Each composable node release involves a release of (at-least) the following components,
+
+1. The Composable Node - main node executable
+
+    Node follows [semver](https://semver.org/) versioning with an additional `v` prefix. Eg: `v2.3.1`. 
+
+2. Runtime wasm - Dali, Picasso and Composable at the time of this writing.
+
+    In order to allow clear identification with the native runtime version of each node release the runtime version for each runtime is an integer derived by concatenating all numbers in the node version and adding to 10000. Eg: `v2.5.1` -> `251` -> 251 + 10000 -> (`dali-10251`, `picasso-10251`, `composable-10251`). The 10000 addition is to prevent collision of version numbers of this scheme with earlier versions before the scheme was introduced.
```

Comment:
Why did you choose an addition? Perhaps, concatenation is better, e.g.
```
λ> "10" <> "251"
"10251"

λ> "10" <> "10251"
"1010251"
```
So, we can avoid collisions with future releases.

Reply:
The function specified here is still a "one to one" function(y = x + 10_000) for any node release version(for vA.B.C, x = ABC). Could you qualify why your suggestion is "better"? The function you propose has a downside of producing larger numbers unnecessarily, eg: "10" + "10251" = 1010251 instead of 10000 + 10251 = 20251.

Reply:
Resolving as this scheme is modified now